gdal/frmts/grib/GNUmakefile

```diff
@@ -14,6 +14,15 @@ ifeq ($(HAVE_JASPER),yes)
 EXTRAFLAGS	:= 	$(EXTRAFLAGS) -DHAVE_JASPER
 endif
 
+ifeq ($(PNG_SETTING),internal)
+EXTRAFLAGS      :=      $(EXTRAFLAGS) -I ../png/libpng -DUSE_PNG
+OBJ     :=      $(OBJ) pngpack.o pngunpack.o enc_png.o dec_png.o
+endif
+
+ifeq ($(LIBZ_SETTING),internal)
```

Comment:
Are you sure you need this zlib thing here ? This is probably only needed for the PNG driver itself